src/assets/css/twitterwall.css

```diff
@@ -63,10 +65,10 @@
 }
 
 .animation{
-	animation:aayush 2s;
-	-webkit-animation:aayush 2s;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The property -moz-animation is compatible with -webkit-animation and should be included as well.](https://www.codacy.com/app/triplez/open-event-webapp/file/2415038484/issues/source?bid=3233857&fileBranchId=3235518#l69)